src/renderer/components/configure-recording-set-page/index.tsx

```diff
@@ -48,10 +50,11 @@ const BUILT_IN_RECORDING_LISTS: BuiltInRecordingList[] = [
 ];
 
 const ConfigureRecordingSetPage: FC<{
+  onSelect: MouseEventHandler<HTMLElement>;
```

Comment:
Let's extract these props into an interface, and document what each prop is.

Reply:
Also, `onSelect` is too vague... Select what?

Maybe something like `onSettingsButtonClick` or just `goToSettingsPage`.

Reply:
fixed